src/ros_filter_bias_estimator.cpp

```diff
@@ -139,10 +147,15 @@ void RosFilterBiasEstimator::updateBiasEstimate(Eigen::Vector3d &orientation_mea
                             orientation_offset_variance_[axis] = alpha_ * orientation_offset_variance_[axis] +
                                 (1.0 - alpha_) * (uncorrected_orientation_variance_[axis] + measurement_variance[axis]);
                         }
+                        // Handle the initial delay - may be immediate if no delay set
+                        initial_delay_met_ |= ((time_s - start_time_) > initial_delay_);
```

Comment:
In case `initial_delay_` is exactly 0, would you want to use a `>=` check here instead?

Reply:
We could. I typically don't do a precise = check for doubles, but this one probably would match. I'll add it.